src/SFA.DAS.EmployerFinance.Web/Views/Transfers/Index.cshtml

```diff
@@ -8,23 +8,26 @@
     ViewBag.ZenDeskLabel = "eas-finance";
 }
 
-<h1 class="heading-xlarge">Transfers</h1>
-<div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-        <h2 class="govuk-heading-m">My pledges</h2>
-        <p>Create a public funding pledge which is shown online so that businesses can apply to you for a transfer of funds.</p>
-        <div class="panel panel-border-wide">
-            You have <strong>@Model.Data.PledgesCount</strong> transfer pledges.
-        </div>
-        <div class="">
-            @if (Model.Data.RenderCreateTransfersPledgeButton)
-            {
-                <a class="button" href="@Url.LevyMatchingTransfersAction("pledges/create/inform")" id="CreateTransfersPledgeButton">Create a transfers pledge</a>
-            }
-            <a href="@Url.LevyMatchingTransfersAction("pledges")" class="button button-secondary">View my transfer pledges and applications</a>
+@if (Model.Data.CanViewTransfersSection)
```

Comment:
Looks like this is hiding the content of the entire page. From my understanding of the acceptance criteria we should just be hiding the 'My pledges' section. If this is the case we should probably rename the property to CanViewPledgesSection too.

Reply:
moved the h1 and renamed the property as we discussed